api/pkg/cli/cmd/info/info.go

```diff
@@ -172,11 +206,88 @@ func (opts *options) run() error {
 	return printer.New(out).Tabbed(tmpl, tmplData)
 }
 
+func (opts *options) AskCatalogName() (string, error) {
```

Comment:
Line no 200-207 won't get be executed now right?

Reply:
I think yes it won't be executed but in that case we don't need to even check if the version is empty, should we remove the condition to check if the version is empty ??